src/strife/p_setup.c

```diff
@@ -510,6 +510,10 @@ void P_LoadBlockMap (int lump)
 
     lumplen = W_LumpLength(lump);
     count = lumplen / 2;
+    if(count >= 32766)
+    {
+	I_Error("Blockmap too large");
+    }
```

Comment:
Respect the code style. For example, put a space between `if` and `(`. Also, don't use tabs for indentation. 

Code style: https://github.com/chocolate-doom/chocolate-doom/blob/master/HACKING.md

Reply:
+1